product_template_multi_link/tests/test_product_template_linker.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import TransactionCase
+
+
+class TestProductTemplateLinker(TransactionCase):
```

Comment:
pls:

1. use SavepointCase
2. move `setUp` code to `setUpClass`
3. use `cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))` after calling super

:)